examples/jupyter/model.xml

```diff
@@ -0,0 +1,1071 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This XML file is temporary and needs to be removed. Best also add a `.gitignore` under the `examples` folder to ignore `*.xml` files.